schema-registry-mock/src/test/java/com/bakdata/schemaregistrymock/ProtobufRegistryMockTest.java

```diff
@@ -19,16 +19,15 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-public class ProtobufRegistryMockTest {
-    private final SchemaRegistryMock schemaRegistry = new SchemaRegistryMock(List.of(new ProtobufSchemaProvider()));
-    private final ParsedSchema schema = createSchema();
+class ProtobufRegistryMockTest {
+    private final SchemaRegistryMock schemaRegistry;
+    private final ParsedSchema schema;
 
-    public ProtobufRegistryMockTest() throws IOException {
-    }
-
-    private static ProtobufSchema createSchema() throws IOException {
+    ProtobufRegistryMockTest() throws IOException {
+        this.schemaRegistry = new SchemaRegistryMock(List.of(new ProtobufSchemaProvider()));
+        
         final Path resources = Path.of("src", "test", "resources");
-        return new ProtobufSchema(Files.readString(resources.resolve("record.proto")));
+        this.schema = new ProtobufSchema(Files.readString(resources.resolve("record.proto")));
```

Comment:
It is always better to use `ProtobufRegistryMockTest.class.getResourceAsStream("/record.proto")` than using direct file paths